src/style/global.css

```
@@ -332,13 +332,45 @@
 }
 
+@keyframes summary-title-gradient {
+  0% {
+    background-position: 100% 50%;
+  }
+  100% {
+    background-position: 0% 50%;
+  }
+}
+
+@utility animate-summary-title-gradient {
```

Stylelint (17.13.0)

[error] Unexpected unknown at-rule "`@utility`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)